src/Bicep.Core.Samples/Files/InvalidExpressions_LF/Completions/azFunctions.json

```diff
@@ -39,6 +39,9 @@
     "textEdit": {
       "range": {},
       "newText": "extensionResourceId($0)"
+    },
+    "command": {
+      "command": "editor.action.triggerParameterHints"
```

Comment:
>triggerParameterHints [](start = 32, length = 21)

This is how we can automatically trigger signature help when a function completion is committed. The command ID is unfortunately editor-specific. It should degrade gracefully in other editors and in the future, we could add command configuration so the IDs can be adjusted to match the editors, so I'm fine with the limitation given the experience improvement. (I've also seen some discussions on the LSP repo about client-agnostic command IDs, but I'm not sure it had any traction.)